client/src/components/auth/login.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+import swal from 'sweetalert';
+import axios from 'axios';
+import {
+  Box, FormLabel, Input, Typography,
+} from '@mui/material';
+import 'typeface-mulish';
+import ButtonComponent from '../Button';
+
+const Login: React.FC = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  type sendUserDataType = () => void;
+
+  const sendUserData: sendUserDataType = async () => {
+    if (email && password) {
+      const userData = {
+        email,
+        password,
+      };
+      const loginResult = await axios.post('/auth/login', userData);
+      if (loginResult.data.message !== 'successful') {
+        swal('email or password is invalid');
+      }
+    } else {
+      swal('In order to login, all of these inputs have to be filled');
+    }
+  };
```

Comment:
wrap your code in try and catch, so you can catch the error message sent from the server to show it to the user.
since if there is any error then your function will not continue to line 23.